Maskbook/Scene/App/FIleService/Model/FileServiceUploadingItem.swift

```diff
@@ -53,16 +55,15 @@ struct FileServiceUploadingItem: Hashable {
     let provider: String
     let state: State
     let content: Data
+    let totalBytes: Double
     let uploadedBytes: Double
     let uploadDate: Date?
     let fileType: ItemType
 
     let mime: String?
     let tx: FileServiceTranscation?
-
-    var totalBytes: Double {
```

Comment:
Why delete this?

Reply:
Moved to line 58, and become a stored property.